### PR TITLE
Update dockerignore following gitignore

### PR DESCRIPTION
**Pull request by lucafavatella**
Follows https://github.com/aeternity/aeternity/pull/2332

### CODE REVIEW

**Comment by lucafavatella** [comment]
[CI test job failed](https://circleci.com/gh/aeternity/aeternity/23569#tests/containers/1):
```
%%% aecore_sync_SUITE ==> all_nodes.two_nodes.ensure_tx_pools_empty: FAILED
%%% aecore_sync_SUITE ==> {test_case_failed,{retry_exhausted,#{info =>
                                         {391,ensure_tx_pools_n_txs_,0,
                                          [{dev1,aeternity_dev1@localhost},
                                           {dev2,aeternity_dev2@localhost}]},
                                     prev => undefined,retries => 5,
                                     tries => 0}}}
```
[Known](https://www.pivotaltracker.com/story/show/163696685), unrelated.

File: Makefile [hunk 1]
```diff
@@ -242,9 +242,12 @@ eunit-fortuna: internal-build
 
 all-tests: eunit ct
 
-docker:
+docker: dockerignore-check
 	@docker build -t aeternity/aeternity:local .
 
+dockerignore-check: | .gitignore .dockerignore
```
Comment (tolbrino) [inline Makefile]: Why not simply use `.dockerignore: .gitignore` ? That way the target wouldn't be PHONY.

Comment (lucafavatella) [reply Makefile]: I removed the fix, I applied this change.

```
$ make .dockerignore 
make: `.dockerignore' is up to date.
```
exit status zero.

So I plan not to apply this change.

Comment (tolbrino) [reply Makefile]: Right. My comment didn't really make sense in the target context.

**Pull request merged**